packages/alea-frontend/pages/homework/[id].tsx

```diff
@@ -0,0 +1,84 @@
+import { Box, CircularProgress, Typography } from '@mui/material';
+import { getCourseInfo, getUserInfo } from '@stex-react/api';
+import { ServerLinksContext } from '@stex-react/stex-react-renderer';
+import { CourseInfo } from '@stex-react/utils';
+import { NextPage } from 'next';
+import { useRouter } from 'next/router';
+import { useContext, useEffect, useState } from 'react';
+import HomeworkPerformanceTable from '../../components/HomeworkPerformanceTable';
+import { getLocaleObject } from '../../lang/utils';
+import MainLayout from '../../layouts/MainLayout';
+import { CourseHeader } from '../course-home/[courseId]';
+import { ForceFauLogin } from '../../components/ForceFAULogin';
+
+const HomeworkPage: NextPage = () => {
+  const router = useRouter();
+
```

Comment:
Try to make use of empty lines carefully. Use them to separate our logically separate parts of code. Don't add space indiscriminately.

Reply:
Noted, will take care in upcoming PRs.